cli/cli.ts

```diff
@@ -5346,7 +5346,7 @@ export interface SnippetInfo {
 
 export function getSnippets(source: string): SnippetInfo[] {
     let snippets: SnippetInfo[] = []
-    let re = /^`{3}([\S]+)?\s*\n([\s\S]+?)\n`{3}\s*?$/gm;
+    let re = /^`{3}\s*([\S]+)?\s*\n([\s\S]+?)\n`{3}\s*?$/gm;
```

Comment:
```suggestion
    let re = /^`{3}[^\S\r\n]*([\S]+)?\s*\n([\s\S]+?)\n`{3}\s*?$/gm;
```

re: last comment, this should be 'matching any whitespace besides newline / carriage return'

Reply:
you could just match spaces and keep it simpler.